### PR TITLE
[qa] Don't set unknown rpcserialversion

### PR DESCRIPTION
**Pull request by maflcko**
When merged, this will merge #9292 as well.

I created a new pull, so 0.13.2 isn't blocked by this.

### CODE REVIEW

**Comment by luke-jr** [comment]
It looks intentional and desirable to allow setting future serialisation versions. Especially something like 9999 to mean "use the very latest even if not the default".

**Comment by sipa** [comment]
@luke-jr Makes sense, but I was assuming that's what the default was.

**Comment by gmaxwell** [comment]
ACK

**Review by laanwj — APPROVED** [review]

File: src/init.cpp [hunk 1]
```diff
@@ -988,6 +988,9 @@ bool AppInitParameterInteraction()
     if (GetArg("-rpcserialversion", DEFAULT_RPC_SERIALIZE_VERSION) < 0)
         return InitError("rpcserialversion must be non-negative.");
 
+    if (GetArg("-rpcserialversion", DEFAULT_RPC_SERIALIZE_VERSION) > 1)
```
Comment (MarcoFalke) [inline src/init.cpp]: Which would imply the default is not always the max. It is not clear from the comments if this is desired.

**Pull request merged**